src/pfe/portal/routes/projects/remoteBind.route.js

```diff
@@ -334,6 +342,10 @@ async function listFiles(absolutePath, relativePath) {
   return fileList;
 }
 
+function listFilesInDirectory(absolutePathToDirectory) {
```

Comment:
You could have achieved the same effect by setting a default value in listFiles:
```
async function listFiles(absolutePath, relativePath = '') {
```